### DISCUSSION TITLE
Script for multiple server addresses with port in custom address list

### DISCUSSION THREAD

**Discussion by Winnudo** [jamulussoftware/jamulus#1663] [Technical Support & Help] [answered]
We have the server only visible via the custom central server address and also have other rooms set up via port sharing.
How could we make the different rooms all visible among each other on one list in this non-public list? Unfortunately only the selected server room is always visible. Further command lines with "ExecStart" in "jamulus.service" did not work for me.
#/bin/bash
set -e

/usr/local/bin/llcon-jamulus/Jamulus --server --nogui \
--port XXXXX \
--welcomemessage "<h2>You can use html tags if you want to format it</h2>" \

/usr/local/bin/llcon-jamulus/Jamulus --server --nogui \
--port YYYY
--welcomemessage "<h2>You can use html tags if you want to format it</h2>" \

Unfortunately does not work.

Can anyone help me with this?
Thanks in advance.

**Comment by dcorson-ticino-com** [comment, marked as answer]
Hi Winnudo,
Have you set up a custom central server ?
You need to start with that and then register your other servers with that custom central server.
Please see the documentation at Jamulus.io for central servers.
Note also that as of (the upcoming) version 3.8.0 the central servers will be called directory servers.

**Reply by Winnudo** [reply]
Hello,
thank you very much for the reply.
Yes we have set up the custom central server? This is also displayed. When I select an IP:22125 it is also displayed. How can iach make it so that all three rooms are displayed in the list at the same time? 22124, 22125 and 22126.
Best regards
Winnudo

**Reply by dcorson-ticino-com** [reply]
You need to register your 3 servers with your central server, use the -e option.
Note the doc for setting up a central server
https://jamulus.io/wiki/Choosing-a-Server-Type#3-central

The combobox custom central server is to define a custom central server that your client uses.
It is then the server listed under "Custom" you open the "List" in the Connect window. 
The client then shows the servers registered on that central server.  
Your servers will never be shown in the custom central server list.

Please take the time to read through the documentation another time.  It is not that simple.

**Reply by Winnudo** [reply]
Thanks for the info and will read up and try. 
Best regards

**Reply by Winnudo** [reply]
Hi dcorson-ticino-com,
It works, thank you very much for the successful tips.
With the best regards
Winnudo
